### ISSUE TITLE
Error while creating new app with --api flag. 

### ISSUE THREAD

**Issue by alexshk** [rails/rails#53077]
Trying to create new app (rails8.0.0.beta1) with this line: `rails new wb --api`
Error:

```
...
gsub  config/environments/production.rb
      create  db/cable_schema.rb
bin/rails aborted!
NameError: undefined local variable or method `console' for an instance of SolidCable::InstallGenerator (NameError)

Tasks: TOP => solid_cable:install
(See full trace by running task with --trace)
```

Removing `--api` flag leads to successful app generation

**Rails version**: Rails 8.0.0.beta1

**Ruby version**: Ruby 3.3.5
```
RubyGems Environment:
  - RUBYGEMS VERSION: 3.5.20
  - RUBY VERSION: 3.3.5 (2024-09-03 patchlevel 100) [arm64-darwin24]
  - INSTALLATION DIRECTORY: /Users/ash/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0
  - USER INSTALLATION DIRECTORY: /Users/ash/.gem/ruby/3.3.0
  - RUBY EXECUTABLE: /Users/ash/.rbenv/versions/3.3.5/bin/ruby
  - GIT EXECUTABLE: /opt/homebrew/bin/git
  - EXECUTABLE DIRECTORY: /Users/ash/.rbenv/versions/3.3.5/bin
  - SPEC CACHE DIRECTORY: /Users/ash/.gem/specs
  - SYSTEM CONFIGURATION DIRECTORY: /Users/ash/.rbenv/versions/3.3.5/etc
  - RUBYGEMS PLATFORMS:
     - ruby
     - arm64-darwin-24
  - GEM PATHS:
     - /Users/ash/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0
     - /Users/ash/.gem/ruby/3.3.0
  - GEM CONFIGURATION:
     - :update_sources => true
     - :verbose => true
     - :backtrace => true
     - :bulk_threshold => 1000
     - "gem" => "--no-rdoc --no-ri"
  - REMOTE SOURCES:
     - https://rubygems.org/
  - SHELL PATH:
    ...
     - /Users/ash/.rbenv/versions/3.3.5/bin
     - /Users/ash/.config/yarn/global/node_modules/.bin
     - /Users/ash/.rbenv/shims
     - /Users/ash/.rbenv/bin
     ...
~
```

**Comment by jeromedalbert** [comment]
There is a [fix](https://github.com/rails/solid_cable/pull/31) on solid_cable's main branch, so cutting a new version of the solid_cable should fix the issue.

cc @morgoth @npezza93

**Comment by npezza93** [comment]
I'll cut a release today

**Comment by npezza93** [comment]
released 3.0.1 which should fix this error.

**Comment by skipkayhil** [comment]
I can confirm that `rails new wb --api` no longer raises an error, thanks for the quick fix!
